R/tidy.R

```diff
@@ -169,9 +169,55 @@ tidy_block = function(
   if (length(exprs) == 0) return(character(0))
   exprs = if (arrow) replace_assignment(exprs) else as.list(exprs)
   deparse = if (inherits(width, 'AsIs')) deparse2 else base::deparse
+
   unlist(lapply(exprs, function(e) {
     x = deparse(e, width)
     x = reindent_lines(x, indent)
+    x = unlist(lapply(
```

Comment:
I'd move this part into a separate function like the previous `reflow_comments()`.